packages/ethereum-contracts/contracts/superfluid/SuperToken.sol

```diff
@@ -143,8 +143,6 @@ contract SuperToken is
 
         address operator = msg.sender;
 
```

Comment:
Worth a comment or link to our technical note (https://github.com/superfluid-finance/protocol-monorepo/wiki/About-ERC-777 to be filled better)